l3build-arguments.lua

```diff
@@ -158,39 +158,38 @@ option_list =
 
 -- This is done as a function (rather than do ... end) as it allows early
 -- termination (break)
-local function argparse()
-  local result = { }
-  local names  = { }
-  local long_options =  { }
-  local short_options = { }
+A.argparse = function (arg)
+  local result = {}
```

Comment:
I take it you don't like the `{ }` convention :)

Reply:
Shorter, less eye movement, better reading...
Seriously, I wanted to discard the change but missed the step...